src/Microsoft.ML/CSharpApi.cs

```diff
@@ -8839,7 +8839,7 @@ public OnlineGradientDescentRegressorPipelineStep(Output output)
     namespace Trainers
     {
 
-        /// <include file='../Microsoft.ML.StandardLearners/Standard/doc.xml' path='doc/members/member[@name="OLS"]/*' />
+        /// <include file='../Microsoft.ML.AdditionalLearners/Standard/doc.xml' path='doc/members/member[@name="OLS"]/*' />
         public sealed partial class OrdinaryLeastSquaresRegressor : Microsoft.ML.Runtime.EntryPoints.CommonInputs.ITrainerInputWithWeight, Microsoft.ML.Runtime.EntryPoints.CommonInputs.ITrainerInputWithLabel, Microsoft.ML.Runtime.EntryPoints.CommonInputs.ITrainerInput, Microsoft.ML.ILearningPipelineItem
```

Comment:
>OrdinaryLeastSquaresRegressor [](start = 36, length = 29)

Hmmm this should not be here either. Nor should LightGBM since it lives in a diff package. Opening an issue about it.

Reply:
Logged issue #618 to document this. 

---
In reply to: [206351985](https://github.com/dotnet/machinelearning/pull/611#discussion_r206351985) [](ancestors = 206351985)